template.dd

```diff
@@ -980,6 +993,10 @@ $(H2 $(LNAME2 function-templates, Function Templates))
         from the function arguments.
     )
 
+    $(P The template type parameter deduction from function arguments is called
```

Comment:
s/The template type parameter deduction from function arguments/The process of deducing template type parameters from function arguments/